src/wallet/rpcwallet.cpp

```diff
@@ -2880,6 +2880,9 @@ static RPCHelpMan listunspent()
                             {RPCResult::Type::STR, "scriptPubKey", "the script key"},
                             {RPCResult::Type::STR_AMOUNT, "amount", "the transaction output amount in " + CURRENCY_UNIT},
                             {RPCResult::Type::NUM, "confirmations", "The number of confirmations"},
+                            {RPCResult::Type::NUM, "ancestorcount", /* optional */ true, "The number of in-mempool ancestor transactions, including this one (if transaction is in the mempool)"},
+                            {RPCResult::Type::NUM, "ancestorsize", /* optional */ true, "The virtual transaction size of in-mempool ancestors, including this one (if transaction is in the mempool)"},
+                            {RPCResult::Type::STR_AMOUNT, "ancestorfees", /* optional */ true, "The total fees of in-mempool ancestors (including this one) with fee deltas used for mining priority in " + CURRENCY_ATOM + " (if transaction is in the mempool)"},
```

Comment:
When testing this, I can see `"ancestorfees": 164` in my output. Is it correct to mark the result type as `RPCResult::Type::STR_AMOUNT` here? I would probably expect `RPCResult::Type::NUM` given that the amount is in satoshis.

More complete output on my machine:

```
[
  {
    "txid": "<txid>",
    "vout": 1,
    "address": "<address>",
    "label": "",
    "scriptPubKey": "<spk>",
    "amount": 0.00100000,
    "confirmations": 0,
    "ancestorcount": 1,
    "ancestorsize": 164,
    "ancestorfees": 164,
    "spendable": true,
    "solvable": true,
    "desc": "<desc>",
    "safe": false
  }
]
```